tests/copylat.cpp

```diff
@@ -69,13 +87,26 @@ int main(int argc, char *argv[])
             case 'r':
                 num_read_iters = strtol(optarg, NULL, 0);
                 break;
+            case 'a':
+                if (strcmp(optarg, "cuMemAlloc") == 0) {
+                    galloc_fn = gpu_mem_alloc;
+                    gfree_fn = gpu_mem_free;
+                }
+                else if (strcmp(optarg, "cuMemCreate") == 0) {
```

Comment:
what if CUDA_VERSION < 10.2? do you return an error or you simply do not accept this option here?

Reply:
In my opinion, returning an error would make the behavior more consistent.
- Compile with `CUDA_VERSION < 10.2`
- Compile with `CUDA_VERSION >= 10.2` but run with `CUDA_VERSION < 10.2`.

On the other hand, we should waive vmm-related unit tests if we don't support vmm. I have just implemented the runtime vmm support checking. Can I ask you to review?